docs/core/porting/cppcli.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Tools for porting to .NET Core
```

Comment:
```suggestion
How to port a C++/CLI project to .NET Core
```

Reply:
It's actually beneficial from an SEO perspective to use different terms in the metadata title than the H1 title.

Reply:
Sounds good.

Reply:
I'll update this to be more accurate while still being a little different from the H1.